util/envDynawo.sh

```diff
@@ -518,6 +716,9 @@ build_3rd_party() {
 # Build a speficic version of third party libraries
 build_3rd_party_version() {
   if ! is_3rd_party_version_installed; then
+    if [ ! -d "$DYNAWO_SRC_DIR/3rdParty" ]; then
+      error_exit "$DYNAWO_SRC_DIR/3rdParty does not exist."
```

Comment:
Really useful?

Reply:
Not really but if somehow the 3rdParty directory is deleted the cd below fails. We could also do
cd $DYNAWO_SRC_DIR/3rdParty || error_exit "$DYNAWO_SRC_DIR/3rdParty does not exist."